router/server.go

```diff
@@ -455,6 +457,15 @@ func copyheadersOut(srcHost string, src ms_http.Header, dstHost string, dst http
 	}
 }
 
+//copy all headers from src to dst
```

Comment:
Little things here:
Always leave a space after `//`
The norm in Go is to write copy(dst, src). See https://golang.org/pkg/builtin/#copy
Function declaration can be written as `copyHeaders(dst, src http.Header)`